apps/backend/lib/src/pagination.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response, NextFunction } from "express";
```

Comment:
If we're just using the express types, why are we importing the full react dependency?

Reply:
Few reasons
1. The intention is to have tests for lib, which will require express (and did when it was implemented). The reason we don't is because we're still in the middle of migrating the test framework
2. It literally doesn't matter because lib isn't built separately from the microservices which use it, and obviously they all use express, so no extra build files are created anyway